python/servo/util.py

```diff
@@ -25,29 +25,8 @@
 from io import BufferedIOBase, BytesIO
 from socket import error as socket_error
 
-try:
-    from ssl import HAS_SNI
-except ImportError:
-    HAS_SNI = False
-
 SCRIPT_PATH = os.path.abspath(os.path.dirname(__file__))
 SERVO_ROOT = os.path.abspath(os.path.join(SCRIPT_PATH, "..", ".."))
-HAS_SNI_AND_RECENT_PYTHON = HAS_SNI and sys.version_info >= (2, 7, 9)
-
-
-def get_static_rust_lang_org_dist():
-    if HAS_SNI_AND_RECENT_PYTHON:
-        return "https://static.rust-lang.org/dist"
-
-    return "https://static-rust-lang-org.s3.amazonaws.com/dist"
-
-
-def get_urlopen_kwargs():
-    # The cafile parameter was added in 2.7.9
-    if HAS_SNI_AND_RECENT_PYTHON:
-        import certifi
```

Comment:
Should we remove `certifi` from [requirements.txt](https://github.com/servo/servo/blob/master/python/requirements.txt#L31)?